src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1843,7 +1843,7 @@ internal enum ErrorCode
         ERR_ExpressionTreeContainsWithExpression = 8849,
         ERR_BadRecordDeclaration = 8850,
 
-        // Available = 8851,
+        WRN_OnlyOneOfGetHashCodeAndEqualsIsDefined = 8851,
```

Comment:
Is it okay to use this id? or I should go to the end of the existing codes?

Reply:
This ID is available to use. This works